tests/e2e/util.go

```diff
@@ -799,16 +799,19 @@ func deleteResourceQuota(client clientset.Interface, namespace string) {
 }
 
 // setResourceQuota resource quota to the specified limit for the given namespace
-func setResourceQuota(client clientset.Interface, namespace string, size string, scName string) {
+func setResourceQuota(client clientset.Interface, namespace string, size string) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 	existingResourceQuota, err := client.CoreV1().ResourceQuotas(namespace).Get(ctx, namespace, metav1.GetOptions{})
```

Comment:
What about the quotas created by `createResourceQuota()`, this has to cleared as well? If a test panics midway and leave the test quota behind, better to clear those as well.

Reply:
yes, I have called the method `deleteResourceQuota` for the same and validated it by terminating the test forcefully and retriggered the tests so that existing quota gets cleared. 
Tests logs uploaded for the same.